src/meta/test/meta_app_operation_test.cpp

```diff
@@ -104,6 +105,31 @@ class meta_app_operation_test : public meta_test_base
 
     void clear_nodes() { _ss->_nodes.clear(); }
 
+    configuration_get_max_replica_count_response get_max_replica_count(const std::string &app_name)
+    {
+        auto req = dsn::make_unique<configuration_get_max_replica_count_request>();
+        req->__set_app_name(app_name);
+
+        configuration_get_max_replica_count_rpc rpc(std::move(req), RPC_CM_GET_MAX_REPLICA_COUNT);
+        _ss->get_max_replica_count(rpc);
+        _ss->wait_all_task();
+
+        return rpc.response();
+    }
+
+    void set_partition_max_replica_count(const std::string &app_name,
+                                         int32_t partition_index,
+                                         int32_t max_replica_count)
+    {
+        dsn::zauto_write_lock l(_ss->_lock);
+
+        auto app = _ss->get_app(app_name);
```

Comment:
L124- L126 can be simplified, you can use function `find_app` in `meta_test_base.cpp`